rules/integrations/azure/credential_access_azure_entra_totp_brute_force_attempts.toml

```diff
@@ -0,0 +1,158 @@
+[metadata]
+creation_date = "2024/12/11"
+integration = ["azure"]
+maturity = "production"
+min_stack_comments = "ES|QL not available until 8.13.0 in technical preview."
+min_stack_version = "8.13.0"
```

Comment:
Do we need this?

Reply:
I believe so. 

https://github.com/elastic/detection-rules/blob/3fa33492163a212918a039f882f3d23c5e195ce3/detection_rules/etc/stack-schema-map.yaml#L90

https://github.com/elastic/detection-rules/blob/3fa33492163a212918a039f882f3d23c5e195ce3/.github/workflows/lock-versions.yml#L9

We may only release from 8.15+, but still support back to 8.12. So no min-stack would backport an ES|QL rule to 8.12.